<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>751</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0.715909, y1:0.182, x2:0.721591, y2:0, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));font: 25pt &quot;Plantagenet Cherokee&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Underwater Image Enhancement&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="statusTip">
     <string extracomment="Start/Stop Camera "/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0.715909, y1:0.182, x2:0.721591, y2:0, stop:0 rgba(221, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProcessButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0.715909, y1:0.182, x2:0.721591, y2:0, stop:0 rgba(221, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Enhance</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageToShow">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>641</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
